resilience4j-micrometer/src/main/java/io/github/resilience4j/micrometer/tagged/CircuitBreakerMetricNames.java

```diff
@@ -17,6 +17,7 @@
     public static final String DEFAULT_CIRCUIT_BREAKER_SLOW_CALL_RATE =
         DEFAULT_PREFIX + ".slow.call.rate";
     private String callsMetricName = DEFAULT_CIRCUIT_BREAKER_CALLS;
+    private String notPermittedCallsMetricName = callsMetricName;
```

Comment:
Per my comment above, I'd strongly suggest we rename this default.

Reply:
Yes, I agree. People can still change it to the old value to be backwards compatible

Reply:
Renamed